pkg/util/mount/nsenter_mount.go

```diff
@@ -188,13 +190,15 @@ func (n *NsenterMounter) IsLikelyNotMountPoint(file string) (bool, error) {
 		// It's safer to assume that it's not a mount point.
 		return true, nil
 	}
-	strOut := strings.TrimSuffix(string(out), "\n")
+	strOut := strings.Split(string(out), " ")[0]
```

Comment:
@jingxu97 Jing, When we split the string we won't need to do a Trim i think. but you will have to confirm what you see in your environment :) LGTM otherwise.